### PR TITLE
Add implementation for Ischemic Heart Disease and Ischemic Stroke

### PR DESCRIPTION
**Pull request by KjellSwedin**
Add the Ischemic Heart Disease and Ischemic Stroke implementations

Update setup.py to pull in recent framework changes.
Rework some of the data_keys naming.
Update models.py to reflect the added models.
Changes to loader.py to work around location in the artifact.

Artifacts build and simulations run.

### CODE REVIEW

File: src/vivarium_nih_us_cvd/components/disease.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+from vivarium_nih_us_cvd.constants import data_keys, models, models
+
+if typing.TYPE_CHECKING:
+    from vivarium.framework.engine import Builder
+    from vivarium.framework.population import SimulantData
+    from vivarium.framework.event import Event
+
+
+class RateTransition(RateTransition_):
```
Comment (rmudambi) [inline src/vivarium_nih_us_cvd/components/disease.py]: You shouldn't need this anymore. You can just use the `RateTransition` and `DiseaseState` straight from vph.

File: src/vivarium_nih_us_cvd/constants/data_keys.py [hunk 2]
```diff
@@ -31,20 +30,24 @@ def log_name(self):
...
+    MI_ACUTE_EMR: str = f'cause.{mod.ACUTE_MI_STATE_NAME}.excess_mortality_rate'
+    MI_POST_EMR: str = f'cause.{mod.POST_MI_STATE_NAME}.excess_mortality_rate'
+    ANGINA_EMR: str = f'cause.{mod.ANGINA_STATE_NAME}.excess_mortality_rate'
+
+    MI_ACUTE_DW: str = f'cause.{mod.ACUTE_MI_STATE_NAME}.disability_weight'
+    MI_POST_DW: str = f'cause.{mod.POST_MI_STATE_NAME}.disability_weight'
+    ANGINA_DW: str = f'cause.{mod.ANGINA_STATE_NAME}.disability_weight'
 
     @property
     def name(self):
```
Comment (rmudambi) [inline src/vivarium_nih_us_cvd/constants/data_keys.py:53]: Maybe set `name = mod.IHD_MODEL_NAME`? Same for Stroke.

File: src/vivarium_nih_us_cvd/components/disease.py (hunk 1 above)
Comment (KjellSwedin) [reply src/vivarium_nih_us_cvd/components/disease.py]: Thanks, pulled it.

File: src/vivarium_nih_us_cvd/constants/data_keys.py (hunk 2 above)
Comment (KjellSwedin) [reply src/vivarium_nih_us_cvd/constants/data_keys.py:53]: Done.

**Review by rmudambi — APPROVED** [review]

**Pull request merged**